Sources/Environment.swift

```diff
@@ -25,9 +25,9 @@ public extension Extension {
   }
 }
 
-public func stencilSwiftEnvironment() -> Environment {
+public func stencilSwiftEnvironment(_ configure: (Extension) -> () = { _ in }) -> Environment {
```

Comment:
Allright, so if it's immutable, modify this function to accept an existing list of extensions (default to empty array). Adding this closure to configure an extension just seems unnecessary and awkward.